src/components/button/button-test.stories.mdx

```diff
@@ -502,87 +502,89 @@ export const FullWidthButtonsStory = () => {
 ### Primary buttons icons before
 
 <Preview>
-  <Story name="primary buttons icons before">
+  <Story name="primary buttons icons before" parameters={{ themeSelector: { chromaticTheme : "sage"}}}>
```

Comment:
Why do we just want the `sage` theme snapshotted for these stories?

Reply:
Unfortunately Chromatic fails to load all the themes, I guess it takes too long to load or the screenshot it too big. Chromatic just reports "failed to render". I chose `sage` theme because it currently falls back to `mint` so it'll look as it does today.

The other stories should surface any bugs introduced to the mint/aegean themes.